apps/microtvm/arduino/template_project/microtvm_api_server.py

```diff
@@ -28,7 +28,7 @@
 from string import Template
 from packaging import version
 
-from tvm.micro.project_api import server
+import server
```

Comment:
Could you please clarify why this change is necessary? As a consequence now `server.py` is being copied to the new project dir too right?

Reply:
Yes, we are copying server.py as well to remove dependency of project api server to TVM.

Reply:
hm I don't follow it. Isn't the client (client.py) always use too when connecting to the server and client is actually called from the tvm.micro.project_api anyways?

Reply:
client starts the microtvm API server, previously it was using sys.executable to directly run microtvm_api_server.py file, however that mean it was using a global python ENV which is the same for any project api server. To distinguish between different project servers and their python requirements we have an alternative to run a bash script to start the sever. This was originally part of the project API design but never used.

Reply:
@mehrdadh I understand all that and I agree that this was exactly the originally need for the project API client/server design. What I was questioning here is why it was necessary to use `server.py` since it was common/generic code for the server in TVM. But I see now that `microtvm_api_server.py`  imports `server.py` also for isolated env, so the python env in this case doesn't have tvm.micro in its env. ok.